clang/include/clang/Basic/BuiltinsSPIRVCommon.td

```diff
@@ -21,3 +21,4 @@ def subgroup_local_invocation_id : SPIRVBuiltin<"uint32_t()", [NoThrow, Const]>;
 def distance : SPIRVBuiltin<"void(...)", [NoThrow, Const]>;
 def length : SPIRVBuiltin<"void(...)", [NoThrow, Const]>;
 def smoothstep : SPIRVBuiltin<"void(...)", [NoThrow, Const, CustomTypeChecking]>;
+def fwidth : SPIRVBuiltin<"void(...)", [NoThrow, Const, CustomTypeChecking]>;
```

Comment:
we only put things here if this builtin should work for both CL and VK contexts. If this is just a VK feature please move to 
`clang/include/clang/Basic/BuiltinsSPIRVVK.td`

As it stands I don't see fwidth in opencl https://registry.khronos.org/OpenCL/specs/3.0-unified/html/OpenCL_C.html#math-functions so I think this will likely need to move.

I only see it in opengl/vk context: https://registry.khronos.org/OpenGL-Refpages/gl4/html/fwidth.xhtml